python/paddle/fluid/distribute_transpiler.py

```diff
@@ -222,8 +222,14 @@ def transpile(self,
 
         # step1: For large parameters and gradients, split them into smaller
         # blocks.
-        param_list = [pg[0] for pg in params_grads]
-        grad_list = [pg[1] for pg in params_grads]
+        param_list = []
```

Comment:
Maybe we can use `filter` for the shorter code:
```python
param_list = filter(lambda x[0] : type(x[0]) == parameter and x[0].trainable == True , params_grads)
```

Reply:
Well, this way actually run 2 for loops and current implementation just run 1for loop